zsh/.zgen-setup

```diff
@@ -7,12 +7,14 @@
 # BSD licensed, see LICENSE.txt
 
 # Clone zgen if you haven't already
-if [[ ! -f ~/zgen/zgen.zsh ]]; then
-  pushd ~
+ZGEN_PARENT_DIR=$HOME/github
+if [[ ! -f $ZGEN_PARENT_DIR/zgen/zgen.zsh ]]; then
+  pushd $ZGEN_PARENT_DIR
```

Comment:
Good point!
I think you don't like this syntax:
`[[ -d "$ZGEN_PARENT_DIR" ]] || mkdir -p $ZGEN_PARENT_DIR`